src/nlsolve/nlsolve.jl

```diff
@@ -42,6 +42,10 @@ function nlsolve!(nlsolver::AbstractNLSolver, integrator)
     apply_step!(nlsolver, integrator)
 
     # check for convergence
+    if iter == 1 && ndz < 1e-5
```

Comment:
was lower needed for this value?

Reply:
Lowering this value makes all convergence tests pass.

Reply:
We can tune this value in subsequent PRs.